eng/scripts/ChangedFiles-Functions.ps1

```diff
@@ -18,8 +18,8 @@ function Get-ChangedFiles($baseCommitish = "HEAD^", $targetCommitish = "HEAD", $
   return $changedFiles
 }
 
-function Get-ChangedSwaggerFiles() {
-  $changedFiles = Get-ChangedFilesUnderSpecification
+function Get-ChangedSwaggerFiles($changedFiles = (Get-ChangedFiles)) {
+  $changedFiles = Get-ChangedFilesUnderSpecification($changedFiles)
```

Comment:
You don't want to put `()` around this as it changes the type here. It might actually work in this case because it is turning it into an array but you should just do `Get-ChangedFilesUnderSpecification $changedFiles`

Reply:
Fix: https://github.com/Azure/azure-rest-api-specs/pull/27425